### DISCUSSION TITLE
不可删除和串改的触发记录

### DISCUSSION THREAD

**Discussion by mike59881051** [gkd-kit/gkd#600] [Ideas]
随着官方规则停更，是否考虑将触发记录设置为不可更改的，以提高安全性，毕竟订阅的野生规则没可能一条条看的非常仔细

**Comment by lisonge** [comment]
> 不可删除和串改的触发记录

不好意思没看懂，我寻思这点击记录除你自己还有谁能改？

---

> 以提高安全性，毕竟订阅的野生规则没可能一条条看的非常仔细

添加订阅时有一个弹窗，你点击 是 才会添加，这时候已经代表你**信任**这个订阅

否则就不要添加未知来源的订阅，你可以使用 <https://github.com/gkd-kit/subscription-template> 自己维护订阅
